batch_job/src/work_queue_factory.c

```diff
@@ -108,6 +107,10 @@ static int64_t factory_timeout = 0;
 
 struct batch_queue *queue = 0;
 
+static const char *mesos_master = NULL;
```

Comment:
I believe nick means if they need to be static global, rather than regular local to main.